pkg/console/operator/sync_v400.go

```diff
@@ -458,10 +458,14 @@ func (c *consoleOperator) ValidateCustomLogo(operatorConfig *operatorsv1.Console
 		return false, "", nil
 	}
 	logoConfigMap, err := c.configMapClient.ConfigMaps(api.OpenShiftConfigNamespace).Get(logoConfigMapName, metav1.GetOptions{})
-	// If we 404, the logo file may not have been created yet.
+	// We should not mount until the configMap with the custom logo is created by user
```

Comment:
So, we can't mount until the `Destination` configmap exists.  We need both!  Source to even test, but destination is the one we are mounting.

```golang
// source exists, great, we can continue, but not yet mount. 
c.configMapClient.ConfigMaps(api.OpenShiftConfigNamespace).Get(logoConfigMapName, metav1.GetOptions{})

// destination CM in openshift-console namespace, great, now we can mount!
logoConfigMap, err := c.configMapClient.ConfigMaps(api.OpenShiftConsoleNamespace).Get(api.OpenShiftCustomLogoConfigMapName, metav1.GetOptions{})
```

Reply:
Ah, per this: 
https://github.com/openshift/console-operator/pull/284/files#diff-283b3e79dd03261002107f949ad39c38R480

Reply:
so the problem is when the user defines the custom logo and also creates the ConfigMap in the `openshift-config` NS for the first time. With the check of the synced CM in the `openshift-console` NS, it wont get ever synced, since we will return `okToMount` with `false`. Will need to updated the logic in https://github.com/openshift/console-operator/blob/master/pkg/console/operator/sync_v400.go#L417 and add that case.

Reply:
Perhaps we need an `okToSync` bool?